packages/ux-capture/src/View.js

```diff
@@ -1,52 +1,53 @@
 import Zone from './Zone';
-
 /**
  * A `View` is a collection of zones representing a single
  * page view or interactive view.
  *
  * Refer to ux-capture README glossary for view definitions.
  * @see: https://github.com/ux-capture/ux-capture#glossary
  */
-function View(props) {
-	this.props = props;
-	this.expectedZones = this.setZones(this.props.zoneConfigs);
+class View {
```

Comment:
Is refactoring this file necessary for adding types?

Reply:
No my typescript plugin seems to be applying auto suggestions without prompting me first